packages/docutils/lib/fs.ts

```diff
@@ -165,12 +166,12 @@ export const readPackageJson = _.memoize(_readPkgJson);
  * blocking operation.
  */
 export const readTypedocJson = _.memoize((typedocJsonPath: string) => {
-  const app = new Application();
-  app.options.setValue('plugin', 'none');
-  app.options.setValue('logger', 'none');
-  app.options.addReader(new TypeDocReader());
-  app.bootstrap({options: path.dirname(typedocJsonPath)});
-  return app.options.getRawValues();
+  // this is a null logger
+  const logger = new Logger();
+  const opts = new Options(logger);
+  // this mutates `opts`
+  new TypeDocReader().read(opts, logger, path.dirname(typedocJsonPath));
+  return opts.getRawValues();
```

Comment:
the perf increase here is probably negligible, but this code is more direct anyway.